public/js/sortfilter.js

```diff
@@ -84,6 +84,16 @@ function filterApply(card, filter) {
   if (filter.category == 'name') {
     res = card.details.name_lower.indexOf(filter.arg) > -1;
   }
+  if (filter.category == 'status') {
+    res = card.status.toLowerCase() === filter.arg;
+  }
+  if (filter.category == 'tag') {
+    var lowerTags = [];
+    card.tags.forEach(function(tag) {
+      lowerTags.push(tag.toLowerCase());
+    });
+    res = lowerTags.indexOf(filter.arg) > -1;
```

Comment:
Rather than instantiate a new variable, you could just use:

If you want to match partial tags, i.e. `tok` matches `token`
```javascript
res = card.tags.some(element => {
    element.toLowerCase().contains(filter.arg);
});
```
If you want to match only complete tags:
```javascript
res = card.tags.some(element => {
    element.toLowerCase() === filter.arg;
});```